### PR TITLE
Onboarding RTL support

### PR DESCRIPTION
**Pull request by yosilevy**
Added RTL calculation in on boarding.

Not too sure on the addition to hass-translation.ts - this can be moved elsewhere maybe directly in to the mixin... It's only for onboarding so that's not a big deal I guess....

### CODE REVIEW

File: src/auth/ha-authorize.ts [hunk 1]
```diff
@@ -71,33 +71,35 @@ class HaAuthorize extends litLocalizeLiteMixin(LitElement) {
...
-        ? html`
-            <ha-pick-auth-provider
-              .resources="${this.resources}"
-              .clientId="${this.clientId}"
-              .authProviders="${inactiveProviders}"
-              @pick-auth-provider="${this._handleAuthProviderPick}"
-            ></ha-pick-auth-provider>
-          `
-        : ""}
+      <div dir="${this.isRTL ? "RTL" : "LTR"}">
```
Comment (bramkragten) [inline src/auth/ha-authorize.ts:74]: We should not wrap it in a div, but just put it on the element (or the body) itself

Comment (yosilevy) [reply src/auth/ha-authorize.ts]: Fixed

**Comment by yosilevy** [comment]
I'd like to also fix this in the actual ha-onboarding but it's pending that problem with the rebuild discussed on Discord

File: src/mixins/lit-localize-lite-mixin.ts [hunk 2]
```diff
@@ -74,6 +80,7 @@ export const litLocalizeLiteMixin = <T extends Constructor<LitElement>>(
       this.resources = {
         [language]: data,
       };
+      this.rtl = await isRTL(this.language!);
```
Comment (bramkragten) [inline src/mixins/lit-localize-lite-mixin.ts:83]: If we would set the `dir` attribute on the parent element, I think we can drop all other code in this PR.

Comment (yosilevy) [reply src/mixins/lit-localize-lite-mixin.ts:83]: The actual <ha-onboarding element is rendered in the HTML template so I'm not sure what to modify there if that's what you mean.
Also the element does not render any wrapper. Add a div?

Comment (bramkragten) [reply src/mixins/lit-localize-lite-mixin.ts:83]: Just use `this.setAttribute("dir", dir);`

**Comment by zsarnett** [comment]
Any progress here?

**Comment by bramkragten** [comment]
Closing this as it has gone stale. Please reopen when you are ready to work on this again.

**Pull request closed without merging**